pkg/plugin/predicates/predicate_manager.go

```diff
@@ -75,12 +74,7 @@ func (p *predicateManagerImpl) predicatesReserve(pod *v1.Pod, node *framework.No
 func (p *predicateManagerImpl) predicatesAllocate(pod *v1.Pod, node *framework.NodeInfo) (plugin string, error error) {
 	ctx := context.Background()
 	state := framework.NewCycleState()
-	plugin, err := p.podFitsNode(ctx, state, *p.allocationPreFilters, *p.allocationFilters, pod, node)
-	if err != nil {
-		events.GetRecorder().Eventf(pod.DeepCopy(), nil, v1.EventTypeWarning,
-			"FailedScheduling", "FailedScheduling", "predicate is not satisfied, error: %s", err.Error())
-	}
-	return plugin, err
+	return p.podFitsNode(ctx, state, *p.allocationPreFilters, *p.allocationFilters, pod, node)
```

Comment:
Should we log this as debug statement?

Reply:
We don't bother to log this for reservations, I don't really see a need to log here either. This can quickly get overwhelming as this can happen multiple times per second and is of little value.